features/git-town-completions/shell-completions.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Rendering Shell autocomplete definitions
+
+  As a shell user
+  I want to use Git Town with autocompletion
+  So that I can use this tool productively.
+
+  Scenario: loading autocompletion in fish
+    Given a fish shell
+    And I run "git-town completions fish | source"
```

Comment:
I think you are right. We cannot really test this. I would add that it is not the responsibility of our test suite to verify whether Cobra's completion stuff works correctly. They have their own tests for that. For example, [here](https://github.com/spf13/cobra/blob/master/zsh_completions_test.go) is their test for the Zsh completion. If there was a bug in Cobra's completion code, it should break their test suite, not ours. The fix should happen in their codebase. The responsibility of our test suite is to test our business logic. This includes that we call the Cobra completion generator properly. I would just check that it prints a piece of the expected output like `sync:Updates the current branch with all relevant changes` or `function _git-town_sync`.